cmd/soroban-rpc/internal/preflight/preflight.go

```diff
@@ -188,26 +194,26 @@ func getInvokeHostFunctionPreflight(params PreflightParameters) (Preflight, erro
 	if err != nil {
 		return Preflight{}, err
 	}
-	minTempEntryExpiration := uint32(0)
-	minPersistentEntryExpiration := uint32(0)
-	maxEntryExpiration := uint32(0)
-	if hasConfig {
-		setting := stateExpirationConfig.Data.MustConfigSetting().MustStateExpirationSettings()
-		minTempEntryExpiration = uint32(setting.MinTempEntryExpiration)
-		minPersistentEntryExpiration = uint32(setting.MinPersistentEntryExpiration)
-		maxEntryExpiration = uint32(setting.MaxEntryExpiration)
+	if !hasConfig {
+		return Preflight{}, errors.New("state expiration config setting missing in ledger storage")
```

Comment:
maybe we use a default here, instead?

Reply:
The preflight library needs a gazillion other parameters without which cannot function properly. The network should be creating this config parameters on every reset.

Reply:
Fair enough!